.github/workflows/tests-3.5.2.yaml

```diff
@@ -0,0 +1,40 @@
+name: Run tests (CPython 3.5.2)
+
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  run_tox:
+    name: tox -e py35 (3.5.2 on ubuntu-16.04)
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - name: Run tox
+        uses: docker://ubuntu:16.04
+        env:
+          DEBIAN_FRONTEND: noninteractive
+        with:
+          # NOTE(vytas): We upgrade pip in the venv in two steps, because 8.1.1
+          #   does not properly resolve python_requires.
+          args: |
+            /bin/bash -c "
+            apt-get update &&
+            apt-get install -y python3.5 python3-venv &&
```

Comment:
shouldn't we specify 3.5.2 here?

Reply:
Ubuntu 16.04 is not updated anymore, we can IMHO just rely on the knowledge that the version is exactly 3.5.2.

Reply:
strange that they did not update it though. 16.04 was supported until 6 month ago, did they not update py3.5?

it log seems to indicate it's 3.5.2 even if apt seems a bit confused:
```
Preparing to unpack .../python3_3.5.1-3_amd64.deb ...
Unpacking python3 (3.5.1-3) ...
```